package.json

```diff
@@ -5,7 +5,8 @@
   "main": "config.js",
   "scripts": {
     "test": "echo \"Error: no test specified\" && exit 1",
-    "start": "node server.js"
+    "start": "node server.js",
+    "dev": "nodemon server.js"
```

Comment:
Added this and nodemon to your devDependencies for an easier time working with it.  You don't need to globally install nodemon if you haven't already - just run `npm run dev`